media-libs/openjpeg/files/openjpeg-2.5.0-r7-off-by-one.patch

```diff
@@ -0,0 +1,40 @@
+From 7bd884f8750892de4f50bf4642fcfbe7011c6bdf Mon Sep 17 00:00:00 2001
```

Comment:
I didn't scrub the patch on purpose - it's literally the output of `wget https://github.com/uclouvain/openjpeg/commit/7bd884f8750892de4f50bf4642fcfbe7011c6bdf.patch` and it applies cleanly, so I thought I shouldn't mess around with it too much. Let me know if I should scrub it.